src/style.css

```diff
@@ -0,0 +1,81 @@
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+label:before {
+  content: "";
+  position: absolute;
+  width: 19px;
+  height: 19px;
+  top: 45px;
+  left: 33px;
+  color: #687c94;
+  background: url(./images/Search.svg);
+}
+
+label:after {
+  content: "";
+  position: absolute;
+  width: 11px;
+  height: 11px;
+  top: 125px;
+  left: 20px;
+  color: #687c94;
+  background: url(./images/Search.svg);
+  background-size: 11px 11px;
+}
+
+.form {
+  font-family: Avenir, sans-serif;
+  width: 100%;
+  font-size: 16px;
+  margin: 12px 0;
+  height: 70px;
+  padding-left: 62px;
+  box-sizing: border-box;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  background-color: #fff;
+}
+
+.form-small {
+  font-family: Avenir, sans-serif;
+  font-size: 14px;
+  margin: 8px 0;
+  height: 42px;
+  padding-left: 33px;
+}
+
+.form:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.form:focus {
+  font-weight: bold;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+  outline: none;
+}
+
+.form-small:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.form-small:focus {
+  font-weight: bold;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border: 1px solid #e1e7ed;
```

Comment:
Move all repeatable colors to CSS variables.

[More info.](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties)